packages/core/admin/_internal/node/staticFiles.ts

```diff
@@ -26,7 +27,7 @@ const getEntryModule = ({ plugins }: EntryModuleArgs): string => {
          * Any modifications made will be discarded.
          */
         ${pluginsImport}
-        import { renderAdmin } from "@strapi/admin/strapi-admin"
+        import { renderAdmin } from "@strapi/strapi/admin"
```

Comment:
@joshuaellis The CI is failing here for `yarn build:size` https://github.com/strapi/strapi/actions/runs/6802714709/job/18519164330#step:6:230, and I see it was also failing on your PR where this change was introduced: https://github.com/strapi/strapi/actions/runs/6744780346/job/18335312467

I am unable to reproduce the error locally. Any idea what might be the issue?

Reply:
Differences in how things are built across the branches - imo we can ignore this & merge. 

If it pops up post merge then lmk and I'll take a look 😎

Reply:
Ok I'll force merge. Can you check on main once it's done please.